projects/packages/protect-models/src/class-vulnerability-model.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Model class for vulnerability data.
+ *
+ * @package automattic/jetpack-protect-models
+ */
+
+namespace Automattic\Jetpack\Protect_Models;
+
+use Automattic\Jetpack\Redirect;
+
+/**
+ * Model class for vulnerability data.
+ */
+class Vulnerability_Model {
+	/**
+	 * Threat ID.
+	 *
+	 * @var null|string
+	 */
+	public $id;
+
+	/**
+	 * Threat Title.
```

Comment:
```suggestion
	 * Vulnerability Title.
```

I suspect this model was copied over from the `Threat_Model`, we should update the documentation to reflect the variation.